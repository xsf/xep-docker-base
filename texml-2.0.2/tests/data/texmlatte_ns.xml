<TeXML xmlns="http://getfo.sourceforge.net/texml/ns1">
 <cmd name="documentclass" nl2="1">
  <parm>article</parm>
 </cmd>
 <cmd name="title">
  <parm>Edward Weston</parm>
 </cmd>
 <env name="document">
  <cmd name="maketitle" nl2="1"/>
  <cmd name="section*" nl2="1">
   <parm>Some biographical information</parm>
  </cmd>
  Edward Weston
  was born in
  1886.
  Weston
  lived until
  1958.
  <cmd name="par" nl2="1"/>
  Weston was a photographer who pioneered the modern use of photography
  as an art form in the United States.
  <cmd name="section*" nl1="1">
   <parm>Some short quotes from Weston</parm>
  </cmd>
  <env name="enumerate">
   <cmd name="item"/>
   <TeXML ligatures="1">``</TeXML>My own eyes are no more than scouts on a preliminary search,
   for the camera's eye may entirely change my idea.<TeXML ligatures="1">''</TeXML>
   <cmd name="item"/>
   <TeXML ligatures="1">``</TeXML>Ultimately success of failure in photographing people depends
   on the photographer's ability to understand his fellow man.<TeXML ligatures="1">''</TeXML>
  </env>
  <cmd name="section*">
   <parm>A longer quote</parm>
  </cmd>
  <env name="quote">
   <TeXML ligatures="1">``</TeXML>One does not think during creative work, any more than one thinks
  when driving a car. But one has a background of years - learning,
  unlearning, success, failure, dreaming, thinking, experience, all
  this - then the moment of creation, the focusing of all into the
  moment. So I can make 'without thought,' fifteen carefully
  considered negatives, one every fifteen minutes, given material
  with as many possibilities. But there is all the eyes have seen in
  this life to influence me.<TeXML ligatures="1">''</TeXML>
  </env>
 </env>
</TeXML>
