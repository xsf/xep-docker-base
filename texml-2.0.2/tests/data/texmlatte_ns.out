\documentclass{article}
\title{Edward Weston}
\begin{document}
\maketitle{}
\section*{Some biographical information}
Edward Weston
  was born in
  1886.
  Weston
  lived until
  1958. \par{}
Weston was a photographer who pioneered the modern use of photography
  as an art form in the United States.
\section*{Some short quotes from Weston}
\begin{enumerate}
\item{} ``My own eyes are no more than scouts on a preliminary search,
   for the camera's eye may entirely change my idea.'' \item{} ``Ultimately success of failure in photographing people depends
   on the photographer's ability to understand his fellow man.''
\end{enumerate}
\section*{A longer quote}
\begin{quote}
``One does not think during creative work, any more than one thinks
  when driving a car. But one has a background of years - learning,
  unlearning, success, failure, dreaming, thinking, experience, all
  this - then the moment of creation, the focusing of all into the
  moment. So I can make 'without thought,' fifteen carefully
  considered negatives, one every fifteen minutes, given material
  with as many possibilities. But there is all the eyes have seen in
  this life to influence me.''
\end{quote}
\end{document}
